odpc.client/src/components/AddRemoveItems.vue

```diff
@@ -18,15 +18,18 @@
   <details ref="detailsRef" aria-live="polite">
     <summary>Toegevoegde {{ itemNamePlural }}</summary>
 
-    <p v-if="!items.length">Er zijn nog geen {{ itemNamePlural }} toegevoegd.</p>
+    <p v-if="!items.length">Er zijn (nog) geen {{ itemNamePlural }} toegevoegd.</p>
 
     <ul v-else class="reset">
       <li v-for="(linkedItem, index) in items" :key="`${itemsId}-${index}`">
         <button
           type="button"
           :title="`${linkedItem} verwijderen`"
           :aria-label="`${linkedItem} verwijderen`"
-          class="button secondary icon-after xmark"
+          class="button secondary"
+          :class="[!isReadonly && `icon-after xmark`]"
```

Comment:
Ik vind dit netter, wat vind jij?
```suggestion
          :class="['button secondary', { 'icon-after xmark': !isReadonly }]"
```